package.json

```diff
@@ -23,9 +23,10 @@
   },
   "homepage": "https://github.com/developer-delta/halogen#readme",
   "dependencies": {
+    "dotenv": "^10.0.0",
     "express": "^4.17.1",
+    "mongodb": "^4.0.0",
     "nodemon": "^2.0.12",
-    "dotenv": "^10.0.0",
-    "mongodb": "^4.0.0"
+    "webfontloader": "^1.6.28"
```

Comment:
Since we are including our font with a regular `link` tag, we actually don't need the webfontloader, so it can be removed from here

Reply:
![image](https://user-images.githubusercontent.com/40808805/126672020-2540097c-ab35-401b-969e-6cf0fa0fb814.png)

Reply:
removed it